dev-utils/watch_ddb_stream.py

```diff
@@ -44,33 +47,41 @@ def extract_primary_key(item: dict):
     return extract_primary_key
 
 
+def make_item_slicer(key_slicer, attribute_names):
```

Comment:
Do you want types on these params?

Reply:
this is just a utility script that i'm changing to make it a little more useful. I'm not very worried about having this look like production code. Though you're right that the type on the other piece of this is straight-up wrong.

Reply:
Cool --  I figured it was just a hold-over from when the closure was moved to being a builder function, but in case the type checking was important, I wanted to call it out. I didn't realize that this was only a script -- I thought it was test helper code for which the type checking might be relevant when writing the tests... I should have looked at the full file in context rather than just the diffs...